lib/post_message_definition.yml

```diff
@@ -16,6 +16,10 @@ post_messages:
       ping:
         payload:
           <<: *user_session_properties
+      go_back:
+        payload:
+          <<: *user_session_properties
+          can_go_back: boolean
```

Comment:
This is the message that the widget sends to the host _after_ it goes back, right? If so, this should be more like "went back" rather than "can go back".

Reply:
Yeah that fells good.